plugin-machine/che-plugin-machine-ext-client/src/main/java/org/eclipse/che/ide/extension/machine/client/processes/ConsolesPanelPresenter.java

```diff
@@ -92,6 +100,50 @@
     Map<String, TerminalPresenter> terminals;
     Map<String, OutputConsole>     commandConsoles;
 
+    private CommandType sshCommandType = new CommandType() {
+        @Override
+        public String getId() {
+            return "ssh";
+        }
+
+        @Override
+        public String getDisplayName() {
+            return "SSH";
+        }
+
+        @Override
+        public SVGResource getIcon() {
+            return null;
+        }
+
+        @Override
+        public Collection<CommandConfigurationPage<? extends CommandConfiguration>> getConfigurationPages() {
+            return emptyList();
+        }
+
+        @Override
+        public CommandConfigurationFactory<? extends CommandConfiguration> getConfigurationFactory() {
+            return null;
+        }
+
+        @Override
+        public String getCommandTemplate() {
+            return "";
+        }
+
+        @Override
+        public String getPreviewUrlTemplate() {
```

Comment:
It's client command type configuration, we need this method to provide default preview template for each command type.